src/Compute/Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -238,9 +255,6 @@ public override void ExecuteCmdlet()
                                 }
                             }
 
-
-
-
                             var psObject = new PSVirtualMachineScaleSet();
```

Comment:
Still need to dev for the other parameter sets.

Reply:
No. These feature requests do not use that parameter set. Would be overextending the scope of this feature, and runs into many other issues as I have seen.